automatilib/cola/views.py

```diff
@@ -155,6 +155,8 @@ def get(
 
         if user := authenticate(request=request, user_response=authenticated_user):
             LOGGER.info(f"Attempting to log user {user.pk} in")
+            if hasattr(user, "is_staff"):
```

Comment:
We don't want everyone in COLA to be an admin in our apps. They're just user managers in COLA